### PR TITLE
added DecimalToHex

### PR DESCRIPTION
**Pull request by AlexDvorak**
contributes to #14

### CODE REVIEW

**Comment by AlexDvorak** [comment]
github says it’s up to date when i push but it isn’t and luhn.js is not part of this PR

**Comment by AlexDvorak** [comment]
@ms10398 problem resolved

**Comment by christianbender** [comment]
@AlexDvorak ***BUG:***  
```js
console.log(decimalToHex(125)); // should be 7D but it was D
```   

***FIX***

Change your function ```decimalToHex(...)``` this way:
```js
function decimalToHex(num){
    let hex_out = [];
    while(num > 15) {
        hex_out.push(intToHex(num%16));
        num = Math.floor(num / 16);
    }
    
    return intToHex(num) + hex_out.join("");  // BUG FIX
}
```

**Comment by christianbender** [comment]
@AlexDvorak Please seperate your pull request in multilpe PRs. One for each filing.

File: Checksums/luhn.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    for (var i = 14; i > -1; i-=2) {
+        var current_number = parseInt(card_number[i]);
+        if (parseInt(card_number[i])*2 > 9) {
+            sum_of_evens += (i*2)-9;
+        } else {
+            sum_of_evens += i*2;
+        }
+    }
+    return 0 === ((sum_of_odds + sum_of_evens) % 10);
+}
```
Comment (christianbender) [inline Checksums/luhn.js]: Please add a test case with 
```js
console.log(luhn(TESTCASE)); // ==> EXPECTED OUTPUT
```

**Comment by ParthS007** [comment]
This pull request **introduces 1 alert** when merging 97ac1c3d248f52474d6957a8e3cdd7a93aff1961 into 9cabd462ec9638ebd859147264b987f047f9df50 - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/Javascript/rev/pr-4917e9c5bc47b43ceb8a3678d3b1154c5a3e1187)

**new alerts:**

* 1 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**